MobileCenterDistribute/MobileCenterDistribute/Resources/fr.lproj/MobileCenterDistribute.strings

```diff
@@ -3,3 +3,4 @@
 "Download" = "Télécharger";
 "Update available" = "Mise à jourr disponible";
 "No release notes" = "Aucune Release Note n'est fournie pour cette version.";
+"In-app-updates are disabled" = "In-app-updates sont désactivées.";
```

Comment:
Now I know a better one, still not perfect: "Les mise à jour depuis l'application sont désactivées."